website/docs/misc/i18next.md

```diff
@@ -185,37 +218,78 @@ i18next handles plurals differently. It requires a separate key to be defined fo
 ```
 
 ```js
+import i18next from "i18next";
+
 i18next.t("key", { count: 0 }); // -> "items"
 i18next.t("key", { count: 1 }); // -> "item"
 i18next.t("key", { count: 5 }); // -> "items"
 ```
 
 ## Context
 
-By providing a context you can differ translations. Both i18next and Lingui have the context feature to differentiate messages.
+By providing context, you can differentiate translations for the same sentences or provide translators with more details. Both i18next and Lingui have the context feature to differentiate messages. However, Lingui provides an automatic additional "context" by including in the `.po` file the locations where each message is used - i18next can't do this from its plain JSON files.
 
-## Summary
+i18next sample:
+
+```js
+import i18next from "i18next";
+
+i18next.t("Right", { context: "direction" });
+```
+
+Lingui sample:
+
+```js
+import { msg } from "@lingui/core/macro";
+
+msg({
+  message: "Right",
+  context: "direction",
+});
+```
 
-This is a rather short comparison. Both libraries have quite different concepts, but at the same time the core internationalization approaches are similar and use the same background.
+## React Integration
 
-On top of that, [Lingui](https://github.com/lingui/js-lingui):
+Both libraries provide React components for handling translations in React applications. Lingui provides a set of [React Macros](/docs/ref/macro.mdx#react-macros) that simplify writing messages directly in your code. i18next provides a `Trans` component to handle translations in JSX.
+
+i18next sample:
+
+```jsx
+import { Trans } from "react-i18next";
+
+const HelloWorld = () => {
+  return <Trans i18nKey="welcome">Hello World!</Trans>;
+};
+```
+
+Lingui sample:
+
+```jsx
+import { Trans } from "@lingui/react/macro";
+
+const HelloWorld = () => {
+  return <Trans>Hello World!</Trans>;
+};
+```
+
```

Comment:
I think the main diffrence between this two, and key selling point of using macro over runtime jsx elements is ability to use child components in the string and don't split message into chunks. I think it's worth to highlight it here.

```jsx
import { Trans } from "react-i18next";

const HelloWorld = () => {
  return <Trans i18nKey="link.check">Check</Trans>{' '}<a href="#"><Trans i18nKey="link.this">this!</Trans</a>;
};
```

Lingui sample:

```jsx
import { Trans } from "@lingui/react/macro";

const HelloWorld = () => {
  return <Trans>Check <a href="#">this!</a></Trans>;
};
```

Reply:
Based on the documentation (https://react.i18next.com/latest/trans-component), it doesn't seem like it should be concatenated that way like in your example

Reply:
Wow, you are right, I've believed that this is not possible with a plain JSX. Hence created this example.

Now I'm extremely interested how do they do that.